umbraco-cloud/upgrades/minor-upgrades.md

```diff
@@ -33,12 +37,10 @@ Once you have the site running locally, you should push your changes to the repo
 
 The upgrade process left off when it was needing three more steps. These three steps now need to be done manually.
 
-1.  Complete the installer
-
-    To complete the installer, you should visit the site on its url. it will be like \`https://dev-mysite.s1.umbraco.io. This will show you the installer screen, where you should insert your backoffice credentials, and follow the process. It will run through a few steps, and afterward, Umbraco will be updated to the latest version.
-2.  Export the metadata files.
-
-    The second thing you need to do is to regenerate the metadata files used for transferring items like document types, data types, and media types. This is done by accessing the Power tools(Kudu) on the project, opening the cmd prompt, and browse to the wwwroot/data folder. Once there, you need to enter `echo > deploy-export`. This will generate the needed files for the upgraded site to work with Umbraco Deploy.
+1. Complete the installer
+    * To complete the installer, you should visit the site on its URL. it will be like \`https://dev-mysite.s1.umbraco.io. This will show you the installer screen, where you should insert your backoffice credentials, and follow the process. It will run through a few steps, and afterward, Umbraco will be updated to the latest version.
```

Comment:
```suggestion
    * To complete the installer, you should visit the site on its URL. it will be like `https://dev-mysite.s1.umbraco.io`. This will show you the installer screen, where you should insert your backoffice credentials and follow the process. It will run through a few steps, and later Umbraco will be updated to the latest version.
```